support/createThirdPartyLicenses.ts

```diff
@@ -0,0 +1,67 @@
+import dedent from "dedent";
+import { writeFile } from "node:fs/promises";
+import { getProjectLicenses } from "generate-license-file";
+import { format } from "prettier";
+
+(async function(): Promise<void> {
+  console.info("Generating third-party licenses file");
+
+  const { exec } = await import("child_process");
+  const { promisify } = await import("util");
+  const execAsync = promisify(exec);
+
+  try {
+    // we need a local `npm install` for `generate-license-file` to resolve licenses properly
+    await execAsync("npm install --no-workspaces");
+
+    const coveredByEsriLicense = [
+      "@arcgis/components-controllers",
+      "@arcgis/components-utils",
+      "@arcgis/lumina",
+      "@esri/calcite-components",
+      "@esri/calcite-components-react",
+      "@esri/calcite-design-tokens",
+      "@esri/calcite-ui-icons",
+      "@esri/eslint-config-calcite",
+      "@esri/eslint-plugin-calcite-components",
+    ];
+
+    const licenses = await getProjectLicenses(".package.json", {
```

Comment:
Should this be `"./package.json"` or `"package.json"`?

Reply:
Oh, good catch. 🎣 It seems that `generate-license-file`'s file [path logic](https://github.com/TobyAndToby/generate-license-file/blob/main/src/packages/generate-license-file/src/lib/internal/resolveDependencies/resolveNpmDependencies.ts#L22) was unintentionally handling this case.